backend/inspirehep/records/config.py

```diff
@@ -514,7 +524,7 @@
         },
     },
     "records-jobs": {
-        "mostrecent": {"title": "Most Recent", "fields": ["-_created"], "order": 1},
+        "mostrecent": {"title": "Most Recent", "fields": ["-_created"], "order": 1}
```

Comment:
why? :D

Reply:
It's because of Black, its criteria for commas is weird... it puts it here and not above in line 420